azure/style.yml

```diff
@@ -1,22 +1,31 @@
 parameters:
   setup: []
+  smoketests: true
```

Comment:
What is the intention of `smoketests`?

It doesn't look like the meaning I'm used to [which is to determine whether further testing should be done](https://en.wikipedia.org/wiki/Smoke_testing_(software)).  `cargo check` seems like it acts like a smoke tests.

Reply:
Ah, sorry, you're right, smoke test isn't really the right word for this.

The meaning I was going for was something that, when it fails, won't prevent the build, but _will_ show up as a warning for the build (a yellow "CI partially succeeded" badge in Azure Pipelines terms). What do you think a better word for this is?

Reply:
Naming is hard.

You've used `canary` in other places and kind of works but by itself doesn't convey enough.